contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.2.0--3.3.0.sql

```diff
@@ -143,5 +143,85 @@ GRANT SELECT ON sys.shipped_objects_not_in_sys TO PUBLIC;
 -- Please have this be one of the last statements executed in this upgrade script.
 DROP PROCEDURE sys.babelfish_drop_deprecated_object(varchar, varchar, varchar);
 
+
+-- This function performs string rewriting for the full text search CONTAINS predicate
+-- in Babelfish
+
+-- For example, a T-SQL query 
+-- SELECT * FROM t WHERE CONTAINS(txt, '"good old days"')
+-- is rewritten into a Postgres query 
+-- SELECT * FROM t WHERE to_tsvector('fts_contains', txt) @@ to_tsquery('fts_contains', 'good <-> old <-> days')
+-- In particular, the string constant '"good old days"' gets rewritten into 'good <-> old <-> days'
+-- This function performs the string rewriting from '"good old days"' to 'good <-> old <-> days'
+
+CREATE OR REPLACE FUNCTION sys.babelfish_fts_contains_rewrite(IN phrase text)
```

Comment:
Do we have handling for FORMSOF phrases yet? We need to detect such cases, and raise an error if it is not supported yet.

Reply:
Done. We just added handling of unsupported cases.